stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/component_browser/messages/viewholder/OnlyAttachmentsMessagesComponentBrowserFragment.kt

```diff
@@ -0,0 +1,48 @@
+package io.getstream.chat.ui.sample.feature.component_browser.messages.viewholder
+
+import androidx.recyclerview.widget.RecyclerView
+import com.getstream.sdk.chat.adapter.MessageListItem
+import io.getstream.chat.android.client.models.Attachment
+import io.getstream.chat.android.client.models.Message
+import io.getstream.chat.android.ui.messages.adapter.viewholder.OnlyMediaAttachmentsViewHolder
+
+class OnlyAttachmentsMessagesComponentBrowserFragment : BaseMessagesComponentBrowserFragment() {
+
+    override fun createAdapter(): RecyclerView.Adapter<*> {
+        return DefaultAdapter(
+            getDummyDeletedMessagesList(),
+            ::OnlyMediaAttachmentsViewHolder,
+            OnlyMediaAttachmentsViewHolder::bind
+        )
+    }
+
+    private fun getDummyDeletedMessagesList(): List<MessageListItem.MessageItem> {
+        return listOf(
+            MessageListItem.MessageItem(
+                message = Message(attachments = mutableListOf(Attachment(type = "image", imageUrl = "https://us-east.stream-io-cdn.com/47574/images/8afae93f-6470-4850-993b-af23d5879244.STREAM_20201203_113150412_PXL_20201203_0828441.jpg?ro=0&Expires=1608286584&Signature=pzWyMXJ~JNgf9VmEpMU5KUkFELFEPZ~dIoIk2sP1DD1uBFYqO2zg8o~e-7xmNWF1eBF8tekrn8ffI5CyZMqTyREMJ1TLqwgUNdkJtkqIMiGf0pIBeKA0jJdcS9ozkIct9l4wx9K31g3jX~Ik5sl3mLd3K~F70Hg1Fe02xaDIucf0s~7Iunda77D~0kOVE-9EsLJtzJ0~BYM6YBJCPGWICsfjuvCXF4dIloo66UFwCdYvsjayRoYolpFyh-pPCWA2Q5jlxU6IIQKjDSU4~nBWrn0qpengHOIr-GIFylysOSAe823PAsYfI7twLyea5~OQoKeK0XEcEMLDC3jeCrCx6A__&Key-Pair-Id=APKAIHG36VEWPDULE23Q"))),
```

Comment:
Since these URLs have an `Expires` param in them, will they stop working in 10 days or so? Perhaps we should find some more reliable source for sample images.

Reply:
I stored images as drawables